src/app/page.js

```diff
@@ -5,13 +5,15 @@ import { Donation } from "./components/home/Donation";
 import { ApplyNow } from "./components/home/ApplyNow";
 import Ourprograms from "./components/home/Ourprograms";
 import Community from "./components/home/Community";
+import Alumni from "./alumni/page";
 
 const Home = () => (
   <Box width="full">
     <HeroSection />
     <Donation />
     <ApplyNow />
     <Ourprograms />
+    <Alumni/>
```

Comment:
This might be the wrong placement. The Alumni page should be a standalone page and not a subsection of the Homepage